x-pack/plugins/enterprise_search/public/applications/__mocks__/kibana_logic.mock.ts

```diff
@@ -5,9 +5,9 @@
  * 2.0.
  */
 
-import { chartPluginMock } from '../../../../../../src/plugins/charts/public/mocks';
+import { mockHistory } from './react_router_history.mock';
```

Comment:
quick warning about this, this import will still hungry mocks React Router itself (`jest.mock('react-router-dom')`) - but I'm hoping to fix that / prevent bleedthrough between RR & Kibana mocks in an upcoming PR